mlflow/openai/autolog.py

```diff
@@ -214,6 +215,11 @@ def _try_parse_raw_response(response: Any) -> Any:
     return response
 
 
+def _is_response_api(original):
+    class_name, _ = original.__qualname__.split(".")
+    return class_name in ["Responses", "AsyncResponses"]
```

Comment:
why do we check the name?

Reply:
This is to check whether it is patching response api or other Apis like chat completion. they have different response format.

Reply:
can we use `original == `Responses.create?`